Source/PermissionButton.swift

```diff
@@ -450,7 +450,15 @@ open class PermissionButton: UIButton {
     Tells the view that its superview changed.
     */
     open override func didMoveToSuperview() {
-        render(.normal)
```

Comment:
Could you use `render()` here please? I prefer to keep `update()` solely as a public interface.